ClearKeep/Scenes/Register/Views/RegisterContentView.swift

```diff
@@ -28,44 +30,53 @@ struct RegisterContentView: View {
 
 	// MARK: - Body
 	var body: some View {
-		GroupBox(label:
-					Text("Register.Title".localized)
-					.font(AppTheme.shared.fontSet.font(style: .body1))
-					.frame(maxWidth: .infinity, alignment: .leading)
-					.padding(.all, Constants.padding)) {
-			VStack(alignment: .center, spacing: Constants.sapcing) {
-				nomalTextfield
-				secureTexfield
-				button
-			}
-			.frame(maxWidth: .infinity, alignment: .center)
+		VStack(alignment: .center, spacing: Constants.sapcing) {
+			Text("Register.Title".localized)
+			.font(AppTheme.shared.fontSet.font(style: .body1))
+			.frame(maxWidth: .infinity, alignment: .leading)
 			.padding(.all, Constants.padding)
+			nomalTextfield
+			secureTexfield
+			button
 		}
+		.padding(.all, Constants.padding)
+		.background(RoundedRectangle(cornerRadius: Constants.radius).fill(backgroundColorView))
+		.frame(maxWidth: .infinity, alignment: .center)
 	}
 }
 
 // MARK: - Private variables
 private extension RegisterContentView {
-	var backgroundColorView: LinearGradient {
-		LinearGradient(gradient: Gradient(colors: backgroundColorButton), startPoint: .leading, endPoint: .trailing)
+	var backgroundColorButton: LinearGradient {
+		LinearGradient(gradient: Gradient(colors: AppTheme.shared.colorSet.gradientPrimary), startPoint: .leading, endPoint: .trailing)
 	}
-	
-	var backgroundColorButton: [Color] {
-		AppTheme.shared.colorSet.gradientPrimary
+
+	var backgroundColorView: Color {
+		colorScheme == .light ? AppTheme.shared.colorSet.offWhite : AppTheme.shared.colorSet.grey6
 	}
-	
+
 	var foregroundColorWhite: Color {
 		AppTheme.shared.colorSet.offWhite
 	}
-	
+
 	var foregroundColorPrimary: Color {
 		AppTheme.shared.colorSet.primaryDefault
 	}
```

Comment:
Don't create new variable if not have any condition.

Reply:
fix complete